### PR TITLE
Add debug feature to show hidden events in timeline

### PR DESCRIPTION
**Pull request by jryans**
This adds a debug feature to show events that would be otherwise be hidden by not having UI to show them. This is nice for working on new event types, etc.

show-hidden-events

### CODE REVIEW

**Comment by turt2live** [comment]
basically fixes https://github.com/vector-im/riot-web/issues/5132 as far as I'm concerned

File: src/settings/Settings.js [hunk 1]
```diff
@@ -130,6 +130,12 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
+    "showHiddenEventsInTimeline": {
+        isFeature: true,
```
Comment (turt2live) [inline src/settings/Settings.js]: technically this isn't a feature, it's a setting. Would recommend applying the same treatment as `enableWidgetScreenshots` to this.

File: src/components/views/messages/ViewSourceEvent.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        return <span className={classes}>
+            {content}
+            <a
+                className="mx_ViewSourceEvent_toggle"
+                href="#"
+                onClick={this.onToggle}
+            />
+        </span>;
+        }
+    }
```
Comment (turt2live) [inline src/components/views/messages/ViewSourceEvent.js]: indentation is off on this line and 1 above

File: src/settings/Settings.js (hunk 1 above)
Comment (jryans) [reply src/settings/Settings.js]: Ah okay, fixed!

File: src/components/views/messages/ViewSourceEvent.js (hunk 2 above)
Comment (jryans) [reply src/components/views/messages/ViewSourceEvent.js]: Thanks, fixed!

**Pull request merged**